features/facebookImport/src/views/explore/details.jsx

```diff
@@ -6,10 +6,11 @@ import "./details.css";
 const ExploreDetails = () => {
     const history = useHistory();
     const { activeAnalysis } = history.location.state;
+    activeAnalysis.setDetailsMode();
```

Comment:
Now I'm _really_ tripping over the name _activeAnalysis_ -> It's _activeStory_, right? :D I'm OK with renaming this later, but it confused me for a moment.